src/obr/OpenFOAM/case.py

```diff
@@ -146,6 +146,22 @@ def is_decomposed(self):
             return False
         return True
 
+    @property
+    def time_folder(self) -> list[Path]:
+        """Returns all timestep folder"""
+
+        def is_time(s: str) -> bool:
+            try:
+                float(s)
+                return True
+            except:
+                return False
+
+        _, fs, _ = next(os.walk(self.path))
+        ret = [self.path / f for f in fs if is_time(f)]
```

Comment:
This is likely unnecessary optimization, but changing the list comprehension to 
`list(filter(lambda f: is_time(f), fs))` runs about twice as fast. 
(in the microsecond range for 10000 elements in `fs`, which is the point that might render it irrelevant.)

In the end, i was just curios about the difference.

Reply:
Wouldn't just `list(filter(is_time, fs))` work as well?

Reply:
Oh yeah, I hadn't considered that. FYI: 
```
In [26]: %timeit [x for x in t if is_time(t)]
4.97 µs ± 17.3 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [27]: %timeit list(filter(lambda x: is_time(x), t))
2.75 µs ± 10.7 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [28]: %timeit list(filter(is_time, t))
2.3 µs ± 13.6 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)```